bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusSlave.java

```diff
@@ -243,6 +244,13 @@ protected void setRegister(Command command, ModbusBindingConfig config)
             } else if (command.equals(OpenClosedType.CLOSED)) {
                 newValue.setValue(0);
             }
+        } else if (command instanceof StopMoveType) {
+            newValue = new SimpleRegister();
+            if (command.equals(StopMoveType.MOVE)) {

```

Comment:
from the shutter point of view it makes indeed sense to trigger for a stop command != 0 since the movement (UP/DOWN) is handled by different type.